scripts/helpers/address_cleaning_inputs.py

```diff
@@ -0,0 +1,103 @@
+"""Subs list for address cleaning"""
+
```

Comment:
In all the regex I would use "\s" instead of " " because it also covers white tabs